everrest-assured/src/main/java/org/everrest/assured/EverrestJetty.java

```diff
@@ -233,7 +233,10 @@ private boolean isRestResource(Class<? extends Object> resourceClass) {
             pDescriptor.accept(rdv);
             return new TestResourceFactory<>(pDescriptor, testObject, field);
         } else if (clazz.getAnnotation(Filter.class) != null &&
-                   (RequestFilter.class.isAssignableFrom(clazz) || ResponseFilter.class.isAssignableFrom(clazz))) {
+                   (RequestFilter.class.isAssignableFrom(clazz)
```

Comment:
Let's remove these checks and then ApplicationPublisher will skip unsupported filters.
WDYT?